rest/data/project.go

```diff
@@ -259,6 +260,27 @@ func (pc *DBProjectConnector) UpdateProjectVars(projectId string, varsModel *res
 	return nil
 }
 
+func (pc *DBProjectConnector) UpdateProjectVarsByValue(toReplace string, replacement string) ([]*db.ChangeInfo, error) {
+	matchingProjects, err := model.GetVarsByValue(toReplace)
+	if matchingProjects == nil {
+		return nil, errors.Wrap(err, "no projects with matching value found")
+	}
+	changes := []*db.ChangeInfo{}
+	for _, project := range matchingProjects {
+		for key, val := range project.Vars {
+			if val == toReplace {
+				project.Vars[key] = replacement
+				change, err := project.Upsert()
+				if err != nil {
+					return nil, errors.Wrapf(err, "problem overwriting variables for project '%s'", project.Id)
```

Comment:
rather than erroring right away when we get to an issue, it may make sense to use a catcher (grip.NewBasicCatcher) and then add each error to the catcher and return the resolved catcher at the end; that way we can return errors with all projects at once.